src/components/avatar/Avatar.tsx

```diff
@@ -5,6 +5,8 @@ interface AvatarProps extends Omit<React.HTMLProps<HTMLDivElement>, 'size'> {
   size: 'xxlarge' | 'xlarge' | 'large' | 'medium' | 'small' | 'xsmall';
   bordered?: boolean;
   variant: 'square' | 'round';
+  /** Border color, default when undefined */
+  borderColor?: 'attention';
```

Comment:
this means the only value possible for borderColor is "attention"

Reply:
Sorry ignores this, it was a WIP that I forgot to remove